src/ontology/edits/oeo-social.omn

```diff
@@ -753,6 +756,89 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/843",
 Class: OEO_00010117
 
     
+Class: OEO_00010122
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "The European Union Emissions Trading System (EU ETS) is policy instrument implementing an emissions trading system covering the European Union, Norway, Iceland and Liechtenstein.",
+        <http://purl.obolibrary.org/obo/IAO_0000118> "EU ETS",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/834
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/857",
+        rdfs:label "European Union Emissions Trading System"@en
+    
+    SubClassOf: 
+        OEO_00140151,
+        OEO_00000503 some OEO_00020064,
+        <http://purl.obolibrary.org/obo/BFO_0000050> some OEO_00010129
+    
+    
+Class: OEO_00010123
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Effort sharing is a policy instrument that sets a common emission reduction goal for a number of countries. Each country has a national reduction goal. Under- or overachievement of that goal can be compensated by trading emission certificates.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/834
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/857",
+        rdfs:label "effort sharing"@en
+    
+    SubClassOf: 
+        OEO_00140151,
+        OEO_00000503 some OEO_00020063
+    
+    
+Class: OEO_00010124
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "The Effort Sharing Decision (ESD) is the European effort sharing for the years 2013-2020. It covers the European Union, Norway, Iceland and Liechtenstein and the following greenhouse gases: carbon dioxide, methane, nitrous oxide, hydrofluorocarbons, perfluorocarbons and sulphur hexafluoride. Emission quantities under the ESD are calculated using global warming potentials from the Fourth IPCC Assessment Report.",
```

Comment:
_The ESD_ looks rather like an individual of `effort sharing` than a universal concept to me?! Same for ESR.

Reply:
I can change that. But in that case, the EU-ETS should be an individual, too.

Reply:
Should `EU climate policy` be an individual, too? If so, the axiom `'EU climate policy' uses some 'EU emission sector division'` would work.

Reply:
Right, seems to me like they should all be individuals.

Reply:
Right, seems to me like they should all be individuals.